### ISSUE TITLE
[@nrwl/storybook] upgrading to Storybook v7 gives configDir error

### ISSUE THREAD

**Issue by dlarroder** [nrwl/nx#14945]
### Current Behavior

Hey. guys, I've just upgraded our version to storybook v7 and I'm getting an error when running it:

```
 >  NX   Required property 'configDir' is missing
```

![Screen Shot 2023-02-13 at 3 26 28 PM](https://user-images.githubusercontent.com/52998821/218396202-993a7983-9b99-4b2b-bbdb-b32cb0db46d2.png)

### Expected Behavior

NX Storybook v7 should run

### Steps to Reproduce

1. Have an existing project with `"@nrwl/storybook": "15.4.5",`
2. Follow the docs here: https://nx.dev/packages/storybook/documents/migrate-storybook-7

### Nx Report

```shell
> nx run ui:build-storybook


>  NX   Required property 'configDir' is missing
```

### Additional Information

NOTE: I'm using @storybook/react-webpack5 in `.storybook/main.js`

```
  framework: {
    name: '@storybook/react-webpack5',
    options: {},
  },
```

**Comment by mandarini** [comment]
@dlarroder thanks for filing this issue!

Question 1: Do you use `nx migrate` to move from one version of Nx to the next?
Question 2: As the docs that you linked say on the top of the page, `Available on Nx v15.5`. If you're using `15.4.5` then it will not work
Question 3: Can I see your `nx report`, and the contents of your `project.json` (specifically the `storybook` and `build-storybook` targets)?

Also, if you have time, can you please provide a reproduction repo?

**Comment by dlarroder** [comment]
Thank you @mandarini! Upgrading NX to a higher version worked! 🙌
